storage/redis/peer_crud.go

```diff
@@ -0,0 +1,79 @@
+package redis
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/chihaya/chihaya/bittorrent"
+	redigo "github.com/garyburd/redigo/redis"
+)
+
+func decodePeerKey(pk string) bittorrent.Peer {
+	return bittorrent.Peer{
+		ID:   bittorrent.PeerIDFromString(string(pk[:20])),
+		Port: binary.BigEndian.Uint16([]byte(pk[20:22])),
+		IP:   net.IP(pk[22:]),
+	}
+}
+
+//Adds an expiry to the set if not refreshed
+func addPeer(s *peerStore, infoHash bittorrent.InfoHash, peerType string,
+	pk serializedPeer) error {
```

Comment:
Pull this up so the function definition is on one line.

Also applies to all other function definitions :)